string affordance_type
geometry_msgs/PoseArray affordance_poses
